IPs/RSKIP157.md

```diff
@@ -12,15 +12,15 @@
 
 ## Abstract
 
-The current RSKIP proposes `cumulativeDifficulty` as a new field to be added in the response of existing JSON-RPC block methods. Such field consists in the sum of a Block's difficulty plus its uncles' difficulties.
+The current RSKIP proposes `cumulativeDifficulty` as a new field to be added in the responses of existing JSON-RPC block methods. The value of this field is the sum of a block's difficulty plus its uncles' difficulties.
 
 ## Motivation
 
-We define the `difficulty` as a magnitude indicating the effort required _for a single miner_ to mine a single block for it to be added to the blockchain. However, this single value does not represent the total effort put by _all miners_ in the network, since the block not only anchors to its parent, but to its uncles as well. We thus define the magnitude `cumulativeDifficulty`, which encompasses not only the block's difficulty, but its mined uncles' difficulty as well.
+We define the `difficulty` as a magnitude indicating the effort required _for a single miner_ to mine a single block for it to be added to the blockchain. However, this single value does not represent the total effort required by _all miners_ in the network, since the block not only anchors to its parent, but to its uncles as well. We thus define the magnitude `cumulativeDifficulty`, which encompasses not only the block's difficulty, but its mined uncles' difficulty as well.
```

Comment:
Alternative to "total effort required" (implies future tense), could be "total effort put in" (past tense).

Reply:
```suggestion
The existing `difficulty` field in responses is defined as a magnitude indicating the effort required _for a single miner_ to mine a single block for it to be added to the blockchain. However, this single value does not represent the total effort required by _all miners_ in the network, since the block not only anchors to its parent, but to its uncles as well. We thus define the magnitude `cumulativeDifficulty`, which encompasses not only the block's difficulty, but its mined uncles' difficulty as well.
```

(highlight that `difficulty` is a field that already exists in the JSON-RPC responses for these methods.)